packages/@lwc/ssr-compiler/src/compile-js/lwc-import.ts

```diff
@@ -35,3 +35,40 @@ export function replaceLwcImport(path: NodePath<ImportDeclaration>, state: Compo
 
     path.replaceWith(b.importDeclaration(path.node.specifiers, b.literal('@lwc/ssr-runtime')));
 }
+
+/**
+ * This handles lwc barrel exports by replacing "lwc" with "@lwc/ssr-runtime"
+ */
+export function replaceNamedLwcExport(path: NodePath<ExportNamedDeclaration>) {
+    if (!path.node || !isLwcSource(path)) {
+        return;
+    }
+
+    path.replaceWith(
+        b.exportNamedDeclaration(
+            path.node.declaration,
+            path.node.specifiers,
```

Comment:
Let's `structuredClone` these for safety. Ditto below for `path.node.exported`.

Reply:
Ok thank you, I updated Dales [replaceLwcImport](https://github.com/salesforce/lwc/pull/5034/files#diff-19dba4b1b2853e94a0315360c33c2db436327b3a19f9c621e786ddb2f45f00aeL36) impl above to keep it consistent.